### PR TITLE
Media Service creation and output of its Primary Key

### PR DESCRIPTION
**Pull request by toto-castaldi**
Media Service creation and output of its Primary Key

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@toto-castaldi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by toto-castaldi** [comment]
Contribution license agreement signed

**Comment by azurecla** [comment]
__@toto-castaldi__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: media-service-output-primary-key/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+    },
+    "variables": {
+        "uniqueResourcePrefix" : "[uniqueString(resourceGroup().id)]",
```
Comment (bmoore-msft) [inline media-service-output-primary-key/azuredeploy.json]: Consider making this a suffix so that resources are easier to find in a list...

**Pull request merged**